eng/scripts/java/Generate.ps1

```diff
@@ -0,0 +1,21 @@
+Set-Location (Resolve-Path (Join-Path $PSScriptRoot '..' '..' '..'))
+
+# Generate
+Remove-Item ./petstore/clients/java/src/main -Recurse -Force
+Set-Location ./petstore/spec
```

Comment:
I think push-location and pop-location could be more convenient here if we want to go into a directory and do something then go back

Reply:
Thanks. Changed.